stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -1,39 +1,41 @@
 const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 
+const winInc = {'bs': 31, 'c4': 23, 'ddg': [1, -5], 'mm': 37, 'rps': 3, 'ttt': 11}
+
 /**
 * Leaderboard information for teams!!!
 * @param {boolean} post if you're trying to store information
-* @param {object} store information to store. Format should be {name: str, info: gameInfo}
+* @param {integer} team which team should increase their score
+* @param {string} game which game was just played
+* param {integer} index which score to use
 * @returns {object} {<game_name>: [red_score, blue_score]}
 */
-module.exports = (post=false, store={}, context, callback) => {
-  lib.utils.storage.get('scores', (err, scores) => {
+module.exports = (post=false, team, game, index=3, context, callback) => {
+  lib.utils.storage.get('scores', 0, (err, scores) => {
     if (err) {
       utils.log.error("error with /scores command", new Error("Accepts error objects"), (err) => {
         return callback(null, 'An error has occurred with your command');
       });
     }
-    if(scores == null) {
+
+    if(scores == null || scores == 0) {
       scores = {};
     }
     if(!post) {
       return callback(null, scores);
     } else {
-      if('name' in store && 'team' in store) {
-        if(!scores.hasOwnProperty(store['name'])) {
-          scores[store['name']] = [0, 0];
-        }
-        if('diff' in store) {
-          scores[store['name']][store['team']] += store['diff'];
-        } else {
-          scores[store['name']][store['team']]++;
-        }
-        lib.utils.storage.set('scores', scores, (err, scores) => {
-          return callback(null, {'success': true});
-        });
+      if (!scores.hasOwnProperty(game)) {
+        scores[game] = [0,0];
+      }
+      if (index != 3) {
```

Comment:
It's unclear what you mean by index. It's a very arbitrary variable name. Maybe winIncIndex would make more sense. Also write better documentation in the docstring for it. I had no clue what it was for until this line. Also make the 3 a constant called NO_INCREMENT or something like that so it's more clear what is happening.

Reply:
It doesn't allow you to put variables as default values in the function declaration.

Reply:
It's still useful to use a constant on this line.

Reply:
👍